examples/forecasting/Forecasting_Task_In_Depth.ipynb

```diff
@@ -0,0 +1,323 @@
+{
```

Comment:
Can this be deleted since we now have in-depth tutorial in docs/tutorials/?

Reply:
will be removed once docs are revised 👍